doc/example/model_selection.ipynb

```diff
@@ -0,0 +1,828 @@
+      "text/plain": [
+       "<Figure size 864x864 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "initial_model_label = {\n",
+    "    initial_model.get_hash(): initial_model.model_id\n",
+    "}\n",
+    "\n",
+    "pvs.plot_history_digraph(\n",
+    "    problem=pypesto_select_problem_2,\n",
+    "    labels={\n",
+    "        **get_digraph_labels(pypesto_select_problem_2.history.values(), criterion=Criterion.AIC),\n",
+    "        **initial_model_label,\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Additional Options\n",
+    "There exist additional options that can be used to further customise selection algorithms.\n",
+    "### Select First Improvement\n",
```

Comment:
not rendered correctly? https://pypesto--397.org.readthedocs.build/en/397/example/model_selection.html#Additional-Options maybe needs preceeding empty lines?